examples/relay-treasurehunt/build/babelGraphQLPlugin.js

```diff
@@ -1,9 +1,4 @@
-var fs = require('fs');
-var getBabelRelayPlugin = require('babel-relay-plugin');
-var path = require('path');
+var getBabelGraphQLPlugin = require('babel-relay-plugin');
```

Comment:
Do we want to use `BabelRelayPlugin` and `BabelGraphQLPlugin` interchangeably? cc @steveluscher @yungsters